# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xfmedia/dbus-0.6.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

i have no idea how correct is this, but most fixes on google looked like this
Alejandro

--- ./src/remote.c.orig	2006-01-31 23:02:50.000000000 -0300
+++ ./src/remote.c	2006-01-31 23:05:47.000000000 -0300
@@ -490,7 +490,7 @@
         g_snprintf(name, 64, XFMEDIA_DBUS_SERVICE_FMT, i);
         dbus_error_init(&derr);
         ret = dbus_bus_request_name(dbus_conn, name,
-                DBUS_NAME_FLAG_PROHIBIT_REPLACEMENT, &derr);
+                DBUS_NAME_FLAG_DO_NOT_QUEUE, &derr);
         if(ret < 0) {
             g_warning("Unable to acquire D-BUS service '%s': %s." \
                 "  Remote control interface will not be available.",
